Fix indentation for in Project approval (#287)
pyansys/dev-guide: doc/source/getting-started/administration.rst

    Once approved, a project can be published to the :ref:`Public PyPI`.

When releasing a project to the public, you should:

* Coordinate with the product line development team, if applicable.
* Maintain the project by means of fixing bugs and providing support for new releases.

> * Uphold Ansys' reputation in the open-source community.

Reviewer: Thanks for the reminder @PipKat .

> * Uphold Ansys' reputation in the open-source community.

Reviewer 2: @akaszynski  I didn't change it to Ansys's reputation--because all of our corporate/marketing stuff uses Ansys' rather than Ansys's. The rule here--as you and I know--is if you say the "s" when forming the possessive of a word ending in "s", then you add the "s". However, I figured I should follow what we do in our corporate communications.

Managerial
^^^^^^^^^^
The managerial part of the approval process ensures that the direct manager,
general managers, vice presidents, or the chief technology officer are aware of
the project's existence and status.

A project must be classified into one of these categories, determining the
number of administrative reviews and approvals needed:

.. grid:: 3
    
    .. grid-item-card:: :octicon:`repo` Documentation projects

>         Documentation projects need direct manager approval, legal review, and

Reviewer 2: ```suggestion
        Documentation projects must have direct manager approval, legal review, and
```

.. card:: |uncheck| The project contains the right licensing.

    * The project has the correct license.
    * The contribution does not contain any strong encryption.
    * Ansys official logos and branding images are used in the project.
    * The Ansys copyright appears in the correct location as required by the
      Legal department.
    * The copyright has the proper formatting, which is ``Copyright (C) YYYY ANSYS, Inc.``.
    * The contribution does not embody any unapproved Ansys intellectual
      property for open sourcing.
    * The contribution does not embody any inventions for which Ansys has
      sought or received patent protection.

>     * Any third-party open-source code included in the contribution has been

Reviewer 2: ```suggestion
    * Any third-party open source code included in the contribution has been
```

      reviewed for security vulnerabilities and includes their license files in
      the repository.

> Open-source dependencies not distributed as part of the project do not need

Reviewer 2: ```suggestion
Open source dependencies not distributed as part of the project do not need
```
Seems like the checker here doesn't like the absence of the hyphen, but we are following Google style guidelines.

their licenses included in the Ansys repository. Examples include dependent

> Node package manager (``npm``) modules or Python packages from PyPI.

Author: https://www.w3schools.com/whatis/whatis_npm.asp has it all capital.

```suggestion
Node Package Manager (``npm``) modules or Python packages from PyPI.
```

Technical
^^^^^^^^^
Technical approval ensures that the project follows the best and latest
software development practices. Request a technical review by sending an email
to `pyansys.core@ansys.com <mailto:pyansys.core@ansys.com>`_.

> The technical review of the project must include the following:

Reviewer 2: ```suggestion
The technical review of the project verifies the following:
```

.. card:: |uncheck| The project contains the right metadata information.
    
    * The project name follows naming conventions.
    * The project version follows :ref:`Semantic versioning`.
    * The project author is ANSYS, Inc.
    * The project maintainer is ANSYS, Inc.

>     * Contact and support information are provided in the project.

Reviewer 2: ```suggestion
    * Contact and support information is provided in the project.
```
Information is the subject, and it is treated as a singular noun in English.
